pkg/reconciler/intevents/pullsubscription/static/pullsubscription_test.go

```diff
@@ -24,14 +24,13 @@ import (
 
 	"cloud.google.com/go/pubsub"
 	"cloud.google.com/go/pubsub/pstest"
+	"google.golang.org/grpc/codes"
```

Comment:
This may be an automated change, but the import seems to fit better where it was previously.